examples/js/mobilenet.js

```diff
@@ -1,89 +1,97 @@
+const Helpers = require('./Helpers');
+
+// Simple configuration, all these files should exist already in the example folder
+const config = {
+   jsonClassIndex: 'imagenet_class_index.json',
+   modelFile: '../models/mobilenet_v2_1.4_224_frozen.pb',
+   inputNode:  'input',
+   outputNode: 'MobilenetV2/Predictions/Reshape_1',
+   imageHeight: 224,
+   imageWidth: 224,
+};
+
+const answerSet = [];
+const helpers  = new Helpers();
+const labels   = JSON.parse(fs.readFileSync(config.jsonClassIndex));
+const redis    = new Redis({ parser: 'javascript' });
+
+let progress = 0;
+
+function run(filenames) {
+
+   /**
+    * We're looping over each image and seeing if we can find a match
+    */
+   filenames.forEach((filename, key, filenames) => {
+      Jimp.read(filename).then((inputImage) => {
+         console.log(`\nWorking on ${filename}`);
+         let image = inputImage.cover(config.imageWidth, config.imageHeight);
+         let normalized = helpers.normalizeRGB(image.bitmap.data, image.hasAlpha());
+         let buffer = Buffer.from(normalized.buffer);
+
+         console.log("...Setting input tensor \n...running model");
+
+         /**
+          * Redis Pipelines, this allows us to set up and batch our commands
+          * If you want to learn more about how IORedis does this check out
+          * https://github.com/luin/ioredis#pipelining
+          */
+         redis.pipeline()
+              .call('AI.MODELSET', 'mobilenet', 'TF', 'CPU', 'INPUTS', config.inputNode, 'OUTPUTS', config.outputNode, buffer)
+              .call('AI.TENSORSET', 'input_' + key, 'FLOAT', 1, config.imageWidth, config.imageHeight, 3, 'BLOB', buffer)
+              /**
+               * Important note: we're using the same input/output keys here...why? Well, in this example
+               * we're not too concerned about anything staying around, we're looking at the images
+               * checking for matches and moving along. In the real world you may need some of
+               * these keys and data to do other things, in which case, you'd want to make
+               * sure you're not overwriting all your keys :)
+               */
+              .call('AI.MODELRUN', 'mobilenet', 'INPUTS', 'input_' + key, 'OUTPUTS', 'output_' + key)
+              .callBuffer('AI.TENSORGET', 'output_' + key, 'BLOB')
+              .exec((err, results) => {
+
+                  /**
+                   * THis looks a wierd, but this is because of how pipelines
+                   * work. When the pipeline is executed the output is
+                   * put into the results as part of an array, so
+                   * in this case, we don't need the output
+                   * from the first or second `.calls` we
+                   * only need the data from the
+                   * AI.TENSORGET() so this
+                   * just helps us get to
+                   * the right data
+                   */
+                  let bufferResults = results[2][1];
```

Comment:
First of all, kudos for the comments you have made. It's so easy to follow now. Thanks a lot.
Here you might have to make the index from `2` to `3` since the `tensorset` is the 4th command in the pipeline